ruleset/rules/0715-solarwinds_rules.xml

```diff
@@ -0,0 +1,150 @@
+
+<!--
+  -  Solarwinds rules
+  -  Created by Wazuh, Inc.
+  -  Copyright (C) 2015-2020, Wazuh Inc.
+  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+-->
+
+<!--
+ID: 66500 - 66999
+-->
+
+<group name="solarwinds_rules">
+  <!-- Sysmon: Library load -->
+  <rule id="66500" level="12">
+    <if_sid>61609</if_sid>
+    <field name="win.eventdata.hashes">SHA256=C15ABAF51E78CA56C0376522D699C978217BF041A3BD3C71D09193EFA5717C71|SHA1=75AF292F34789A1C782EA36C7127BF6106F595E8|MD5=56CEB6D0011D87B6E4D7023D7EF85676</field>
+    <mitre>
+      <id>T1587.001</id>
+      <id>T1587</id>
+      <id>T1588.003</id>
+      <id>T1588</id>
+      <id>T1190</id>
+      <id>T1195</id>
+      <id>T1195.002</id>
+      <id>T1053</id>
+      <id>T1053.005</id>
+      <id>T1036</id>
+      <id>T1036.004</id>
+      <id>T1036.005</id>
+      <id>T1553</id>
+      <id>T1553.002</id>
+    </mitre>
+    <description>Supernova trojan detected in $(win.eventdata.imageLoaded) library.</description>
```

Comment:
I'd set description field after  if_sid tag. This will help to read the target 1st and then the behavior.